src/lib/utils/meta_tables/meta_mock_table.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include "utils/meta_tables/abstract_meta_table.hpp"
+
+namespace opossum {
+
+/**
+ * This is a class that mocks the behavior of a meta table.
+ * It can be useful in tests as it provides how often methods were called.
```

Comment:
Does this have to be in `src/lib`?

Reply:
I took the MockNode as orientation, this can also be in test/utils